src/content/configuration/entry-context.mdx

```diff
@@ -30,7 +30,8 @@ module.exports = {
 };
 ```
 
-By default, the current directory is used, but it's recommended to pass a value in your configuration. This makes your configuration independent from CWD (current working directory).
+By default, the current working directory of the Node.js process is used, but it's recommended to pass a value in your configuration. This makes your configuration 
```

Comment:
```suggestion
By default, the current working directory of the Node.js is used, but it's recommended to pass a value in your configuration. This makes your configuration 
```

Let's remove that `process`, seems terrifying for novice.

Reply:
Okay I've removed